cli/medperf/config.py

```diff
@@ -44,7 +45,7 @@
 # Storage config
 config_storage = Path.home().resolve() / ".medperf_config"
 logs_storage = Path.home().resolve() / ".medperf_logs"
-config_path = str(config_storage / "config.yaml")
+config_path = getenv("MEDPERF_CONFIG_PATH", str(config_storage / "config.yaml"))
```

Comment:
Defining this env var is now required for all medperf installations?

Reply:
No, if env is not defined, default value is used (`str(config_storage / "config.yaml")`). `getenv` works similar to `dict.get()` method